### PR TITLE
RavenDB-16939  - SlowTests.Cluster.ClusterOperationTests.ChangesApiFa…

### PR DESCRIPTION
**Pull request by shiranshalom**
…ilOver

### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-16939/SlowTestsClusterClusterOperationTestsChangesApiFailOver

### Type of change

- Bug fix

### How risky is the change?

- Low 

### Backward compatibility

- Non breaking change

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing 

- It has been verified by manual testing

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

File: test/SlowTests/Cluster/ClusterOperationTests.cs [hunk 1]
```diff
@@ -284,27 +279,40 @@ public async Task ChangesApiFailOver()
...
+                        }, true, interval: 333));
+
 
                     using (var session = store.OpenAsyncSession())
                     {
                         await session.StoreAsync(new User(), "users/1", cts.Token);
                         await session.SaveChangesAsync(cts.Token);
                     }
 
+                    Assert.True(await WaitForChangeVectorInClusterAsync(databaseServers, db, 30_000));
```
Comment (garayx) [inline test/SlowTests/Cluster/ClusterOperationTests.cs]:
```suggestion
                    Assert.True(await WaitForChangeVectorInClusterAsync(databaseServers, db, 30_000), "WaitForChangeVectorInClusterAsync");
```

Comment (shiranshalom) [reply test/SlowTests/Cluster/ClusterOperationTests.cs]: done

File: test/SlowTests/Cluster/ClusterOperationTests.cs [hunk 2]
```diff
@@ -254,20 +248,21 @@ public async Task PreferredNodeShouldBeRestoredAfterBroadcast()
         [Fact]
         public async Task ChangesApiFailOver()
         {
-            using (var cts = new CancellationTokenSource(TimeSpan.FromMinutes(3)))
+            DebuggerAttachedTimeout.DisableLongTimespan = true;
```
Comment (ppekrol) [inline test/SlowTests/Cluster/ClusterOperationTests.cs]: this should be removed?

Comment (shiranshalom) [reply test/SlowTests/Cluster/ClusterOperationTests.cs]: Yep. Done

**Review by garayx — APPROVED** [review]

File: test/SlowTests/Cluster/ClusterOperationTests.cs (hunk 2 above)
Comment (shiranshalom) [reply test/SlowTests/Cluster/ClusterOperationTests.cs]: Oh, oops my mistake. 
The `DebuggerAttachedTimeout.DisableLongTimespan = true;` is fine as far as I know, and now we can remove the `CancellationTokenSource` (what I did in this fix). let me know if you want it back  :-)

Comment (ppekrol) [reply test/SlowTests/Cluster/ClusterOperationTests.cs]: CTS was there to timeout the test if needed, I would not like this test to hang.

Comment (shiranshalom) [reply test/SlowTests/Cluster/ClusterOperationTests.cs]: OK sorry. Done (returned CTS and keeping `DebuggerAttachedTimeout.DisableLongTimespan = true;`)

Comment (ppekrol) [reply test/SlowTests/Cluster/ClusterOperationTests.cs]: But this is static and affects other tests, isnt this for debugging purposes only?

Comment (shiranshalom) [reply test/SlowTests/Cluster/ClusterOperationTests.cs]: Done

**Review by ppekrol — APPROVED** [review]

**Pull request merged**